src/main/scala/com/lucidchart/open/relate/SqlStatementParser.scala

```diff
@@ -101,15 +94,10 @@ private[relate] object SqlStatementParser {
    * @param query the current query in a StringBuilder
    */
   private def insertTuples(numTuples: Int, tupleSize: Int, query: StringBuilder): Unit = {
-    if (numTuples > 0) {
+    for (i <- 0 to numTuples - 1) {
```

Comment:
@msiebert and @tmccombs : I got that, sure you both are right. Thank you! Scala is so good :)